src/ContentHubClient.php

```diff
@@ -281,7 +281,7 @@ public static function clientNameExists(
     $options['body'] = json_encode(['name' => $name]);
     // Attempt to fetch the client name, if it works.
     try {
-      $client->get("settings/clients/$name");
+      $client->get("settings/client/name/$name");
```

Comment:
```suggestion
      $this->getClientByName($name);
```
We have an existing public function for this endpoint, we reuse the same.

Reply:
The function is a static function, should not refer with `$this` in class internal context.

Reply:
Be careful of the context.

Reply:
Ah, I missed `static` scope. Will approve it.
**Side note:** I don't see usage of this static function anywhere.

Reply:
Yeah, that's a different issue, it can be removed in that case, if it is really not used.

Reply:
As discussed, function marked as deprecated.